public/external-text/default.yaml

```diff
@@ -650,51 +656,87 @@ help:
           Frost Degree Days: Degree-days during the specified time 
           period below 0&deg;C.
 
-        ### Return period variables
+        ### Return period/level variables
 
-        A return period variable describes extreme temperature or
+        A return period/level variable (for CMIP5/CMIP6 respectively) describes extreme temperature or
         precipitation events that would be expected to occur once
         during a specified "return period," for example once every
         20 years.
         These datasets are calculated from model output using a
         generalized extreme value distribution.
         This data is available for six thirty year climatological
-        periods from 1960 to 2100, three historical and three
-        projected.
-        This data is available for all Canada.
+        periods, three historical and three projected.
+        For CMIP5, these periods are from 1961-2100, whereas
+        for CMIP6, they are from 1971-2100. This data is available for all Canada.
 
-        * **`rp5pr`**
+        * **`rp5pr`** / **`rl5pr`**
```

Comment:
The description for these variables above says that it's "return period/level", but I could change it to something like "return period/level (rp for CMIP5 and rl for CMIP6 respectively)". Or we could make these variables uppercase, but that may look awkward since the other variables are lowercase in the help.

Reply:
That might help a little, seems worth a shot.